api/base/serializers.py

```diff
@@ -55,27 +56,45 @@ def _url_val(val, obj, serializer, **kwargs):
 
 
 class IDField(ser.CharField):
+    """
+    ID field that validates that 'id' in the request body is the same as the instance 'id' for single requests.
+    """
     def __init__(self, **kwargs):
         kwargs['label'] = 'ID'
         super(IDField, self).__init__(**kwargs)
 
+    #Overrides CharField
     def to_internal_value(self, data):
         update_methods = ['PUT', 'PATCH']
```

Comment:
As you add more batch operations, it's worth defining `update_methods` in one central consistent location. Not huge now, but I see it defined twice so far- worth consolidating if we roll batch operations out to more endpoints.